scripts/check_regression.py

```
@@ -0,0 +1,224 @@
+        """レポートを生成"""
+        report_lines = []
+        report_lines.append("# ベンチマーク解析結果")
+        report_lines.append("")
+        
+        # サマリー
+        total_count = len(results)
+        error_count = sum(1 for r in results if r["status"] == "error")
+        warning_count = sum(1 for r in results if r["status"] == "warning")
+        improved_count = sum(1 for r in results if r["status"] == "improved")
+        
+        report_lines.append(f"## サマリー")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)